packages/cli/internal/pkg/cli/spec/project.go

```diff
@@ -9,3 +11,12 @@ type Project struct {
 	Data          []Data              `yaml:"data,omitempty"`
 	Contexts      map[string]Context  `yaml:"contexts,omitempty"`
 }
+
+func (projectSpec *Project) GetContext(contextName string) (Context, error) {
```

Comment:
Nit, because Go doesn't allow overloaded methods, possibly a better name would be GetContextForName

Reply:
I thought about that but the current standard in the project is to access the fields directly instead of using getters/setters.